js/private/js_run_binary.bzl

```diff
@@ -107,6 +110,21 @@ def js_run_binary(
 
             This makes node binaries match the expected bazel paradigm.
 
+        run_from_execroot: Run the `tool` js_binary entry_point script that is in the execroot of the output tree
+            instead of the copy that is in runfiles.
+
+            Runfiles of `tool` are all hoisted to `srcs` of the underlying `run_binary` so they are included as execroot
```

Comment:
Is that the `srcs` param? So they are hoisted **as** srcs of the underline `run_binary`?

Reply:
Yeah. So they end up in the execroot. Otherwise they won't be present since Bazel only includes the runfiles in the sandbox and not the execroot files that back them.